README.md

```diff
@@ -1,24 +1,200 @@
-# README
+<a name="readme-top"></a>
```

Comment:
**NOTE:** The optional points are only optional because they were not mentioned by the previous reviewer, but I highly recommend that you take them into consideration.

There are a few things you can improve about your README to make it more professional.

## [OPTIONAL] 

- Please remove the Microverse logo and heading. It is meant to be replaced by your project logo if the is one.

![image](https://user-images.githubusercontent.com/38088818/215272536-9d871bb1-8795-4cbc-9d73-33d85f87be77.png)

- The `Show your support` section is not documented. Kindly write the relevant information for this section.

![image](https://user-images.githubusercontent.com/38088818/215269420-b75df450-36fe-4d5f-960c-81f14790bf78.png)

- Surely there's someone you can acknowledge that helped you complete this project whether it's your coding partner(s), the Morning session team for motivation, or the Microverse team for the great curriculum.

![image](https://user-images.githubusercontent.com/38088818/215272422-8aa47b7b-3aa0-444a-ba89-65e87046fda2.png)

- Please make sure that you do not have any broken links on your README. The link to your live demo is broken, since you have not deployed the project yet please leave a text like `coming soon...` instead.

![image](https://user-images.githubusercontent.com/38088818/215272267-864bc46a-3d5a-4373-86a5-8cb1193d8b41.png)


- Kindly remove all the description texts under each section that came with the template, they are meant to guide you on how to modify each section. They should not be in the final documentation of your project.

For example:
![image](https://user-images.githubusercontent.com/38088818/215272145-3dcefbe0-188b-49dc-a260-b184afdc6902.png)
![image](https://user-images.githubusercontent.com/38088818/215272182-c80f9cd8-471e-4ca7-a943-a13480ced91e.png)
![image](https://user-images.githubusercontent.com/38088818/215272213-403935b9-42b0-48af-88f6-4128a6218997.png)

And so forth... Be sure to remove the rest that is not included in the screenshots above.